BlankProject/ContentView.swift

```diff
@@ -8,14 +8,21 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    @State var didTap = false
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            ZStack {
+                didTap ? Color(.red)
+                    .edgesIgnoringSafeArea(.all)
```

Comment:
maybe instead of setting `.edgesIgnoringSafeArea(.all)` everytime the color changes, you can just make do 

`
Color(didTap ? .red : .systemPink).edgesIgnoringSafeArea(.all)
`